pytoda/preprocessing/crawlers.py

```diff
@@ -127,3 +131,58 @@ def get_smiles_from_pubchem(
                 logger.warninig(f'Did not find any result for drug: {drug}')
```

Comment:
```suggestion
                logger.warning(f'Did not find any result for drug: {drug}')
```
fixing same copy paste typo here